### PR TITLE
Add a first version of kanagawa theme

### PR DESCRIPTION
**Pull request by zetashift**
Here's my attempt at an helix version for: https://github.com/rebelot/kanagawa.nvim

image

### CODE REVIEW

File: runtime/themes/kanagawa.toml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+"function.macro" = "waveRed"
+"tag" = "springBlue"
+"namespace" = "surimiOrange"
+"attribute" = "peachRed"
+"constructor" = "springBlue"
+"module" = "waveAqua2"
+"special" = "peachRed"
+
+## Markup modifiers
+"markup.heading.marker" = "fujiGrey"
```
Comment (the-mikedavis) [inline runtime/themes/kanagawa.toml:73]:
```suggestion
"markup.heading.marker" = "fujiGray"
```

Comment (zetashift) [reply runtime/themes/kanagawa.toml]: Oh how embarassing x)! Great catch.

**Review by the-mikedavis — APPROVED** [review]

**Comment by the-mikedavis** [comment]
Could you add a screenshot of this to the themes wiki? https://github.com/helix-editor/helix/wiki/Themes

**Comment by zetashift** [comment]
> Could you add a screenshot of this to the themes wiki? https://github.com/helix-editor/helix/wiki/Themes

I've added a screenshot: https://github.com/helix-editor/helix/wiki/Themes#kanagawa

Thank you very much! :D

**Pull request merged**